cluster-autoscaler/simulator/predicates.go

```diff
@@ -204,6 +204,27 @@ func NewTestPredicateChecker() *PredicateChecker {
 	}
 }
 
+// NewCustomTestPredicateChecker builds test version of PredicateChecker with additional predicates.
+func NewCustomTestPredicateChecker(customPredicates ...predicates.FitPredicate) *PredicateChecker {
+	newPredicates := []predicateInfo{
+		{name: "default", predicate: predicates.GeneralPredicates},
```

Comment:
It looks kinda too arbitrary to me that we have some 2 predicates hard coded and extend it with some extra ones.

Reply:
Is there something special about "default" and "ready" that we always want them?

Reply:
Sure, fixed